README.rst

```diff
@@ -40,11 +40,29 @@ To run the tests::
 
     $ nosetests
 
-To lint your code automatically when you make changes::
+Integration tests are not run by default. To run them, you must set up an
+integration environment and edit the test settings::
+
+    $ cp tests/test_integration/test_settings.py.sample \
+    $    tests/test_integration/test_settings.py
```

Comment:
No need for the prompt (`$`) on this line, as it's a continuation of the previous line